includes/class-gp-notifications.php

```diff
@@ -379,13 +379,14 @@ public static function get_email_body( WP_Comment $comment, array $comment_meta
 		 * @param WP_Comment $comment      The comment object.
 		 * @param array      $comment_meta The meta values for the comment.
 		 */
-		$output  = apply_filters( 'gp_notification_pre_email_body', $output, $comment, $comment_meta );
-		$output .= esc_html__( 'Hi there,', 'glotpress' );
-		$output .= '<br><br>';
-		$url     = GP_Route_Translation_Helpers::get_permalink( $project->path, $original->id );
-		$output .= wp_kses(
+		$output          = apply_filters( 'gp_notification_pre_email_body', $output, $comment, $comment_meta );
+		$output         .= esc_html__( 'Hi there,', 'glotpress' );
+		$output         .= '<br><br>';
+		$url             = GP_Route_Translation_Helpers::get_permalink( $project->path, $original->id );
+		$link_to_comment = $url . '#comment-' . $comment->comment_ID;
```

Comment:
Could we move these two lines before compiling the `$output` variable? Then we won't have all the whitespace changes.